search/plugins/ezplatformsearch/ezplatformsearch.php

```diff
@@ -58,7 +58,12 @@ public function needCommit()
      */
     public function needRemoveWithUpdate()
     {
-        return true;
+        if ( $this->iniConfig->variable( 'IndexOptions', 'NeedRemoveWithUpdate' ) === 'true' )
```

Comment:
Nitpick: Can't you just use `return $this->iniConfig->variable( 'IndexOptions', 'NeedRemoveWithUpdate' ) === 'true');` :)

Reply:
Yup, I amended the commit.